pkg/cmd/roachtest/tests/typeorm.go

```diff
@@ -70,7 +70,7 @@ func registerTypeORM(r registry.Registry) {
 			c,
 			node,
 			"install dependencies",
-			`sudo apt-get install make python3 libpq-dev python-dev gcc g++ `+
+			`sudo apt-get install -y make python3 libpq-dev python-dev gcc g++ `+
```

Comment:
i think we want this on line 91 as well

Reply:
oh and is line 62 already fine? referencing 7552077190c03cdff8f9d0f452007de559ec80e7

Reply:
62 and 91 didn't ask to confirm. 
73 asked to confirm that some space would be needed.

Added to 91 just in case.